src/app/docs/numeric/numeric-docs.component.html

```diff
@@ -6,4 +6,199 @@
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1100 | skyNumeric}}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1500 with zero digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1500 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1500 with one digit:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1500 | skyNumeric}}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1000000 with any digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1000000 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1000000000 with any digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1000000000 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1000000000000 with any digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1000000000000 | skyNumeric:{digits: 0} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1234000 with 2 digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1234000 | skyNumeric:{digits: 2} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1235000000 with 2 digits:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1235000000 | skyNumeric:{digits: 2} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1.45 with 1 digit:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1.45 | skyNumeric}}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1234567 with 1 digit:
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1234567 | skyNumeric}}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            1234567 without truncation
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{1234567 | skyNumeric:{truncate: false} }}
+          </sky-definition-list-value>
+        </p>
+        <p>
+          <sky-definition-list-label>
+            15501 with truncation after 10000 and 1 digit
```

Comment:
I'm not sure I follow what's happening here. Why do we need `truncateAfter: 10000` here? Wouldn't the result have been the same without it? I had assumed that the module tracked places and would have abbreviated 15501 as 15.5K even without being provided a point to truncate after. If not, doesn't that make this harder to use programmatically?

Reply:
Agreed. Not sure why this is here... I removed it.